xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-preferences/xwiki-platform-notifications-preferences-api/pom.xml

```diff
@@ -65,4 +65,44 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.revapi</groupId>
+        <artifactId>revapi-maven-plugin</artifactId>
+        <configuration>
+          <analysisConfiguration>
+            <revapi.differences>
+              <differences>
+                <item>
+                  <ignore>true</ignore>
+                  <code>java.method.addedToInterface</code>
+                  <new>method org.xwiki.notifications.preferences.email.NotificationEmailDiffType org.xwiki.notifications.preferences.email.NotificationEmailUserPreferenceManager::getDiffType(org.xwiki.model.reference.DocumentReference)</new>
```

Comment:
These breaking changes can be avoided by making the new methods `default` in `NotificationEmailUserPreferenceManager`. 

See https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HBackwardCompatibility

Also, will this work be applied on 14.6RC1+ too?